infra/docker-compose/dev/docker-compose.yml

```diff
@@ -35,7 +35,7 @@ services:
     restart: on-failure
     volumes:
       - "kube:/.kube"
-      - "/tmp:/tmp"
+      - "${PWD}:/tmp"
```

Comment:
Why has this been changed?

Reply:
Was testing some configuration, will change it back.